index.js

```diff
@@ -60,6 +60,23 @@ function requestAsEventEmitter(opts) {
 			});
 		});
 
+		// this method is not documented, prints a warning if used while
```

Comment:
[Node.js 0.8.29](https://nodejs.org/docs/v0.8.28/api/http.html#http_response_removeheader_name) have this method - so it is safe to use.